ui/selectors/permissions.js

```diff
@@ -310,7 +310,9 @@ export function getSnapInstallOrUpdateRequests(state) {
   return Object.values(state.metamask.pendingApprovals)
     .filter(
       ({ type }) =>
-        type === 'wallet_installSnap' || type === 'wallet_updateSnap',
+        type === 'wallet_installSnap' ||
+        type === 'wallet_updateSnap' ||
+        type === 'wallet_installSnapResult',
```

Comment:
nit: is there an enum for the rpc methods? maybe we should be using that.

Reply:
There is not, these aren't RPC methods, but approval types. (They do look a lot like RPC methods though 😅 )

Reply:
Haha yes, ok well that's fine then.